### PR TITLE
Web login timeout

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/9901

### CODE REVIEW

File: modules/administration/pages/ssl-certs.adoc [hunk 1]
```diff
@@ -294,3 +310,47 @@ To complete the process, you need to update the database with this command:
...
 If you use bootstrap, remember to also update your bootstrap scripts to reflect the new certificate information.
+
+
+
+== Replace Certificates with a Third Party Certificiate
+
+You can replace active certificates on your {productname} installation with a new third party certificate.
+To replace the certificates, you can replace the installed CA certificate RPM with a new RPM containing the third party certificate, and then update the database.
+
+This procedure is similar to the one described in "Create and Replace CA and Server Certificates" with the difference, that we import the certificates generated by an external PKI.
```
Comment (keichwa) [inline modules/administration/pages/ssl-certs.adoc]:
```suggestion
This procedure is similar to the one described in xref:administration:ssl-certs.adoc#custom-ssl-create-replace-cert[] with the difference, that we import the certificates generated by an external PKI.
```
I'm not sure whether within files we can use the simple `<<custom-ssl-create-replace-cert>>` technique.

File: modules/administration/pages/ssl-certs.adoc [hunk 2]
```diff
@@ -294,3 +310,47 @@ To complete the process, you need to update the database with this command:
...
+This command might end with a warning that the server certificate request file could not be found.
+This file is not required, but you can provide it, by copying it into the server directory as `server.csr`.
++
+----
+cp <Certificate_Request_File>.csr /root/ssl-build/<Server_Name>/server.csr
+----
++
+When you repeat the command to generate the server certificate RPM, the warning should not be shown anymore.
+
+Now you have an ssl-build directory similar to the one generated by "Procedure: Creating New Certificates" from the chapter before.
```
Comment (keichwa) [inline modules/administration/pages/ssl-certs.adoc]: proper xrefs wanted...

File: modules/administration/pages/tshoot-logintimeout.adoc [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+Result: What happens after the user has completed the fix?
+
+If more detailed instructions are required, put them in a "Resolving" procedure:
+.Procedure: Resolving Widget Wobbles
+. First step
+. Another step
+. Last step
+////
+
+By default, the {productname} {webui} will require users to log in again after thirty minutes.
```
Comment (keichwa) [inline modules/administration/pages/tshoot-logintimeout.adoc]:
```suggestion
By default, the {productname} {webui} will require users to log in again after 30{nbsp}minutes.
```

**Comment by Loquacity** [comment]
Something has gone seriously awry here.

**Comment by Loquacity** [comment]
Please review https://github.com/SUSE/doc-susemanager/pull/875 instead.

**Pull request closed without merging**